libs/common-const/src/nativeAndWrappedTokens.ts

```diff
@@ -1,62 +1,84 @@
-import { SupportedChainId, SupportedChainId as ChainId } from '@cowprotocol/cow-sdk'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
 import { TokenWithLogo } from './types'
-import { WETH9 } from '@uniswap/sdk-core'
 import { cowprotocolTokenLogoUrl } from './cowprotocolTokenLogoUrl'
 
-// See https://github.com/cowprotocol/contracts/commit/821b5a8da213297b0f7f1d8b17c893c5627020af#diff-12bbbe13cd5cf42d639e34a39d8795021ba40d3ee1e1a8282df652eb161a11d6R13
-export const NATIVE_CURRENCY_BUY_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
+export const DEFAULT_NATIVE_CURRENCY_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
Why the `DEFAULT_` prefix?
I suggest not having it, since there are no other addresses for native currency AFAICT

Reply:
yep, i agree. Its not expected to be overridden

Reply:
Renamed to `NATIVE_CURRENCY_ADDRESS `